planet/api/experimental/basemaps_client.py

```diff
@@ -71,3 +81,23 @@ def list_quads_in_mosaic(
             for s in quad_list:
                 yield s
             url = resp.json().get('_links', {}).get('_next')
+
+    def get_quads_in_mosaic_for_region(self, mosaic_id, aoi):
+        url = self.base_url + f'basemaps/v1/mosaics/{mosaic_id}/quads/search?minimal=true'
+        resp = self._post(url, aoi).json()
+        return resp['items']
+
+    def get_time_series(self, aoi, start_date, end_date):
+        mosaic_series = self.list_mosaic_series(name__is='Global Monthly')['series'][0]
+        series_id = mosaic_series['id']
+        geometry = aoi[0]['config']
```

Comment:
Hmm, it looks like aoi is not geojson here. What's this config? Can you give an example aoi?